### PR TITLE
Bump to JUnit 5.6.1

### PR DESCRIPTION
**Pull request by gastaldi**

### CODE REVIEW

File: devtools/gradle/build.gradle [hunk 1]
```diff
@@ -30,9 +30,7 @@ dependencies {
 
     testImplementation 'org.mockito:mockito-core:3.3.3'
     testImplementation 'org.assertj:assertj-core:3.15.0'
-    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.6.0'
-    testImplementation 'org.junit.jupiter:junit-jupiter-params:5.6.0'
-    testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.6.0'
+    testImplementation 'org.junit.jupiter:junit-jupiter:5.6.1'
```
Comment (geoand) [inline devtools/gradle/build.gradle]: :+1:

**Comment by geoand** [comment]
How about these also: 

https://github.com/quarkusio/quarkus/blob/master/integration-tests/maven/src/it/setup-on-existing-pom-it/pom.xml#L28

https://github.com/quarkusio/quarkus/blob/master/integration-tests/maven/src/it/setup-with-custom-quarkus-version-it/pom.xml#L27

**Comment by gastaldi** [comment]
Done

**Review by geoand — APPROVED** [review]
:+1:

**Pull request merged**